sources/Blocks/Konomi/build-module/konomi-konomi-block-view.js

```
@@ -0,0 +1 @@
+import*as e from"@wordpress/interactivity";var o={d:(e,t)=>{for(var r in t)o.o(t,r)&&!o.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},o:(e,o)=>Object.prototype.hasOwnProperty.call(e,o)};const t=(r={getContext:()=>e.getContext,getElement:()=>e.getElement,store:()=>e.store,useLayoutEffect:()=>e.useLayoutEffect},n={},o.d(n,r),n);var r,n;function s(e){for(;e&&"konomi"!==e.dataset.wpInteractive;)e=e.parentElement;return e}(0,t.store)("konomi",{state:{},actions:{closeLoginModal:()=>{(0,t.getContext)("konomi").loginRequired=!1}},callbacks:{maybeRenderResponseError:()=>{const e=(0,t.getContext)("konomi");(0,t.useLayoutEffect)((()=>{const o=(0,t.getElement)();var r;o.ref&&e.error.code&&(r=o.ref,(e=>{const o=function(e){var o;return null!==(o=s(e)?.querySelector(".konomi-response-message"))&&void 0!==o?o:null}(e);return o?Promise.resolve(o):Promise.reject(new Error("Konomi: Popover element not found."))})(r).then((e=>(e.showPopover(),new Promise((o=>{setTimeout((()=>{e.hidePopover(),o()}),3e3)})))))).finally((()=>{e.error={code:"",message:""}}))}),[e.error.code,e.error.message])},toggleLoginModal:()=>{const e=(0,t.getElement)();if(!(e.ref instanceof HTMLElement))return;const o=(0,t.getContext)("konomi"),r=function(e){const o=s(e)?.querySelector(".konomi-login-modal");return function(e){if(null==e)throw new Error("Element is not an HTMLDialogElement")}(o),o}(e.ref);o.loginRequired?r.showModal():r.close()}}});
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)